src/main/java/com/maxmind/geoip2/model/IspResponse.java

```diff
@@ -7,20 +7,22 @@
  */
 public class IspResponse extends AbstractResponse {
 
-    @JsonProperty("autonomous_system_number")
-    private Integer autonomousSystemNumber;
+    private final Integer autonomousSystemNumber;
+    private final String autonomousSystemOrganization;
+    private final String isp;
+    private final String organization;
+    private final String ipAddress;
 
-    @JsonProperty("autonomous_system_organization")
```

Comment:
Right, I already did it for a bunch of getters, because I noticed thus because the `JsonTest` was failing. I guess extra tests like this one should be added.